microsoft-edge/webview2/releasenotes.md

```diff
@@ -14,37 +14,51 @@ keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edg
 
 Release notes for [WebView2 SDK][WebView2NuGetGallery].  
 
+## 0.X.XXX
+
+[NuGet package][WebView2NuGetGallery0.X.XXX] | minimum Microsoft Edge version 8X.0.XXX.X.
+
+**Re-compile your app after updating the NuGet package.** 
+
+*   **Announcement:** WebView2 now supports WinForms and WPF on .NET Framework 4.6.2 or later and .NET Core 3.0 or later in the **pre-release package**
+*   Checkout the [WPF Getting Started Guide]() to get started building WPF applications and our [WPF Reference]() for WPF specific APIs
+*   Checkout the [WinForms Getting Started Guide]() to get started building WinForms applications and our [WinForms Reference]() for WinForms specific APIs
+*   Checkout [.NET Reference]() for CoreWebView2 APIs 
+*   **Known Issues:** We are aware of some issues in this pre-release that we will resolve in future releases
+    - **DPI Awareness:** WebView2 for WPF is currently not DPI aware. When initializing WebView2 on high DPI monitors, there is a known issue where the WebView at first initializes as a fraction of the window until the WebView is resized.
+    - **WPF Designer:** This release does not currently support the WPF designer. Place the WebView2 control in your app by directly modifying the appropriate XAML in a text editor
```

Comment:
```suggestion
    - DPI Awareness: WebView2 for WPF is currently not DPI aware. When initializing WebView2 on high DPI monitors, there is a known issue where the WebView initializes as a fraction of the window until the WebView is resized.
    - WPF Designer: This release does not currently support the WPF designer. Place the WebView2 control in your app by directly modifying the appropriate XAML in a text editor
```

Reply:
Hey Reeza we have traditionally bolded these titles so users skimming through the release notes have the ability to see what issues are. Not bolding these specifically would make the entire doc look inconsistent

Reply:
your first bullet point also incorrectly changes the meaning of the the issue

Reply:
Appreciate the comments, but will revert back to the original text